app/config.py

```diff
@@ -28,13 +28,30 @@ class Settings:
     teamname_file: Path = Path("/home/pi/teamname.txt")
     zone: bool = False
 
+    """
```

Comment:
I've tried to follow PEP8 (the python style guide in this). We could and should check this with CI but don't yet: https://peps.python.org/pep-0008/#block-comments

From PEP8 multiline blockcomments in python
 > are indented to the same level as that code. Each line of a block comment starts with a # and a single space (unless it is indented text inside the comment).
>
> Paragraphs inside a block comment are separated by a line containing a single #.

I think the logic of this is that declaring strings in the middle of your code can *sometimes* do things see `__doc__` in https://docs.python.org/3/reference/datamodel.html#the-standard-type-hierarchy

We really should do this in CI shouldn't we